lib/metric-config-parser/metric_config_parser/tests/test_experiment.py

```diff
@@ -376,6 +378,30 @@ def test_sample_size_none_experiment(self, experiments, config_collection):
         cfg = spec.resolve(experiments[7], config_collection)
         assert cfg.experiment.sample_size is None
 
+    def test_analysis_unit_default(self, experiments, config_collection):
+        conf = dedent(
+            """
+            [experiment]
+            enrollment_period = 7
+            """
+        )
+        spec = AnalysisSpec.from_dict(toml.loads(conf))
+        cfg = spec.resolve(experiments[1], config_collection)
+        assert cfg.experiment.randomization_unit is None
+        assert cfg.experiment.analysis_unit == AnalysisUnit.CLIENT
```

Comment:
For comprehensiveness, should we add a test that needs to parse the randomization unit into "normandy" or is that unnecessary?

Reply:
Yea, not a bad idea. Maybe I should rewrite the test as a parametrized thing instead of jamming it into the existing object. I'll do that.

Reply:
Very nice, the new tests look great!